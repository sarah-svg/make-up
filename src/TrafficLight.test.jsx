import React from 'react';
import { render, screen, waitFor, cleanup } from '@testing-library/react';
import TrafficLights from './components/app/TrafficLight';

describe('Page All Artists ', () => {
  afterEach(() => cleanup());

  it('displays all of the artist', () => {
    render(<TrafficLights/>);
    const ul = screen.getByText('Light');

    return waitFor(() => {
      expect(ul).not.toBeEmptyDOMElement();
    });
  });
  it('renders App', () => {
    const { asFragment } = render(<TrafficLights />);
    expect(asFragment()).toMatchSnapshot();
  });

});
