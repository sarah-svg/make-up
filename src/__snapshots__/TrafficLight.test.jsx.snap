// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page All Artists  renders App 1`] = `
<DocumentFragment>
  <div>
    <title>
      Light
    </title>
    <div
      class="black-box"
      date-testid="  Red, Green, yellow"
    >
      <div
        class="light red"
      />
      <div
        class="light inactive"
      />
      <div
        class="light inactive"
      />
      Red
    </div>
  </div>
</DocumentFragment>
`;
